src/components/tutor-schedule/TutorSchedule.styles.ts

```diff
@@ -0,0 +1,89 @@
+export const styles = {
+  general_container: {
+    width: '500px',
+    height: '500px'
+  },
+
+  section_title: {
+    typography: 'h5',
```

Comment:
We thy to use one approach in all styling files with reusable enums

![image](https://github.com/user-attachments/assets/0a05d1ed-6ee4-40fe-ba12-36b008ce47cd)